libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -17,6 +17,9 @@ export enum ButtonSize {
   LG = 'lg',
 }
 
+const ATTENTION_LEVEL_4_DEPRECATION_WARNING =
+  'Deprecation warning: The "kirby-button" support for using input property "attentionLevel" with the value "4" will be removed in a future release of Kirby designsystem.';
```

Comment:
Consider also informing the user that their use of attention level 4 has in fact been remapped to attention level 3.

Reply:
-good point...